core/src/test/scala/unit/kafka/message/MessageTest.scala

```diff
@@ -66,26 +65,31 @@ class MessageTest extends JUnitSuite {
       } else {
         TestUtils.checkEquals(ByteBuffer.wrap(v.payload), v.message.payload)
       }
+
       // check timestamp
-      if (v.magicValue > Message.MagicValue_V0)
+      if (v.magicValue > Message.MagicValue_V0) {
         assertEquals("Timestamp should be the same", v.timestamp, v.message.timestamp)
-       else
+      } else {
         assertEquals("Timestamp should be the NoTimestamp", Message.NoTimestamp, v.message.timestamp)
+      }
```

Comment:
There are differing opinions on this one, so I'd avoid making this change in unrelated code to what the PR is trying to accomplish. To offer a counterpoint, I've never seen a bug caused by this in 12 years.